src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final String DIVIDER = "    ____________________________________________________________";
+    public static final int MAX_CAPACITY = 100;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        runChatBot(MAX_CAPACITY);
+    }
+
+    private static void runChatBot(int MAX_CAPACITY) {
+        Scanner input = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_CAPACITY];
+        taskManager(input, tasks);
+    }
+
+    private static void taskManager(Scanner input, Task[] Tasks) {
+        int taskCounter = 0;
+        while(true) {
+            String userInput = getInput(input.nextLine());
+            String[] userCommands = userInput.split(" ", 2);
+            if (userCommands[0].equalsIgnoreCase("bye")) {
+                printByeMessage();
+                break;
+            } else if (userCommands[0].equalsIgnoreCase("list")) {
+                printList(Tasks, taskCounter);
+            } else if (userCommands[0].equalsIgnoreCase("done")) {
+                printDone(userCommands, Tasks);
+            } else if(userCommands[0].equalsIgnoreCase("todo")) {
+                addTodo(Tasks, taskCounter, userCommands);
+                taskCounter ++;
+            } else if(userCommands[0].equalsIgnoreCase("event")) {
+                addEvent(Tasks, taskCounter, userCommands[1]);
+                taskCounter++;
+            } else if(userCommands[0].equalsIgnoreCase("deadline")) {
```

Comment:
Should there be a spacing between the else if and the '(' as seen in your other else if sections for more consistency?